source/gui/__init__.py

```diff
@@ -85,7 +85,7 @@ def prePopup(self):
 		"""
 		nvdaPid = os.getpid()
 		focus = api.getFocusObject()
-		if focus.processID != nvdaPid:
+		if focus.processID != nvdaPid or getattr(focus, "isPrevFocusOnNvdaPopup", False):

```

Comment:
I think this deserves a comment. What happens without this change?

Reply:
Without this change, the default gestures bound to the scripts on the NVDA Python Console could not be customized through the Input Gestures dialog. These are the first scripts defined on the NVDA AppModule itself.

Comments added as of fff2ee34e, feel free to amend them as you see fit.